bot/arena_bot.py

```diff
@@ -8,8 +8,8 @@
 from discord import Intents
 from discord.ext.commands import Bot
 
-from config import COMMAND_PREFIX
-from database import Database
+from bot.config import COMMAND_PREFIX
+from bot.database import Database
```

Comment:
Why have all the imports for source files been prefixed with `bot.`? I want to avoid this.

Reply:
Ive made the necessary changes